doc/rados/operations/pools.rst

```diff
@@ -466,31 +482,31 @@ You may set values for the following keys:
 
 .. describe:: bulk
 
-   :Description: Sets and unsets the bulk flag on a given pool.
+   :Description: Sets or unsets the ``BULK`` flag on a given pool.
    :Type: Boolean
    :Valid Range: ``true``/``1`` sets flag, ``false``/``0`` unsets flag
 
 .. _write_fadvise_dontneed:
 
 .. describe:: write_fadvise_dontneed
 
-   :Description: Sets and unsets the WRITE_FADVISE_DONTNEED flag on a given pool.
+   :Description: Sets or unsets the ``WRITE_FADVISE_DONTNEED`` flag on a given pool.
    :Type: Integer
    :Valid Range: ``1`` sets flag, ``0`` unsets flag
 
 .. _noscrub:
 
 .. describe:: noscrub
 
-   :Description: Sets and unsets the NOSCRUB flag on a given pool.
+   :Description: Sets or unsets the ``NOSCRUB` flag on a given pool.
```

Comment:
s/``NOSCRUB`/``NOSCRUB``/

(add an extra backtick after "NOSCRUB")

Reply:
Was waiting for the render to check that.  Pushing the fix.